ios/uPortMobile/Classes/EthereumSigner.m

```diff
@@ -0,0 +1,203 @@
+// Copyright (C) 2018 ConsenSys AG
+//
+// This file is part of uPort Mobile App.
+//
+// uPort Mobile App is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// uPort Mobile App is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with uPort Mobile App.  If not, see <http://www.gnu.org/licenses/>.
+
+#import "EthereumSigner.h"
+
+#import "CoreBitcoin/CoreBitcoin+Categories.h"
+#import "CoreBitcoin/BTCKey.h"
+#include <openssl/bn.h>
+#include <openssl/ec.h>
+#include <openssl/obj_mac.h>
+
+NSMutableData *compressedPublicKey(EC_KEY *key) {
+  if (!key) return nil;
```

Comment:
I believe it's good practice to always have the `if`/`else`/`while`/... code blocks as a real block between `{  }`.  Placing code on the same line makes that they can be missed easily (because it's a second way to format these statements; best is to have as little ways to format code as possible making it easier to scan by a human).